migrations/versions/a56c82a8d02e_add_detail_to_checkpoint.py

```diff
@@ -0,0 +1,27 @@
+"""Add detail to checkpoint
+
+Revision ID: a56c82a8d02e
+Revises: f4fd4c80f40a
+Create Date: 2021-01-22 16:35:07.063082
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+revision = 'a56c82a8d02e'
+down_revision = 'f4fd4c80f40a'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    url = op.get_bind().engine.url
+    collation = 'utf8_bin' if 'mysql' in url.drivername else None
+    op.add_column(
+        'commcare_export_runs',
+        sa.Column('data_source', sa.Unicode(255, collation=collation))
+    )
+
+def downgrade():
+    op.drop_column('commcare_export_runs', 'data_source')
```

Comment:
for my own education how is this file generated?

Reply:
```
alembic -c migrations/alembic.ini revision -m "description"
```
(https://github.com/dimagi/commcare-export/blob/master/migrations/README.md)